<ui version="4.0" >
 <class>ListDock</class>
 <widget class="QWidget" name="ListDock" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops" >
   <bool>true</bool>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLineEdit" name="searchString" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="Tags" >
     <property name="text" >
      <string>Tag:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2" >
    <widget class="QComboBox" name="tagsCombo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="toolTip" >
      <string>Filter view by chosing tag</string>
     </property>
     <property name="sizeAdjustPolicy" >
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QWidget" native="1" name="widgetTools" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
     <widget class="QPushButton" name="removeButton" >
      <property name="geometry" >
       <rect>
        <x>80</x>
        <y>0</y>
        <width>30</width>
        <height>29</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>R</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAllButton" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>32</width>
        <height>29</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="statusTip" >
       <string>Show complete collection of available fonts</string>
      </property>
      <property name="text" >
       <string>All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewActivatedButton" >
      <property name="geometry" >
       <rect>
        <x>40</x>
        <y>0</y>
        <width>32</width>
        <height>29</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="statusTip" >
       <string>Show all activated fonts</string>
      </property>
      <property name="text" >
       <string>Act</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLabel" name="setlabel" >
     <property name="text" >
      <string>Tag Set:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QComboBox" name="tagsetCombo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Show only fonts that belong to chosen tagset</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QLabel" name="filtersCrumb" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4" >
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="FMNameList" name="fontTree" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="font" >
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0" >
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="focusPolicy" >
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="acceptDrops" >
       <bool>true</bool>
      </property>
      <property name="autoFillBackground" >
       <bool>false</bool>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth" >
       <number>1</number>
      </property>
      <property name="dragDropMode" >
       <enum>QAbstractItemView::DropOnly</enum>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="textElideMode" >
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="autoExpandDelay" >
       <number>1</number>
      </property>
      <property name="columnCount" >
       <number>2</number>
      </property>
      <column>
       <property name="text" >
        <string>Names</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Files</string>
       </property>
      </column>
     </widget>
     <widget class="FMPreviewList" name="previewList" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FMPreviewList</class>
   <extends>QGraphicsView</extends>
   <header>fmpreviewlist.h</header>
  </customwidget>
  <customwidget>
   <class>FMNameList</class>
   <extends>QTreeWidget</extends>
   <header>fmnamelist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
